localization/android/src/main/java/tonlabs/uikit/localization/UIKitLocalizationModule.java

```diff
@@ -52,6 +53,10 @@ public String getName() {
         dates.put("localePattern", dateInfo.getLocalePattern());
         dates.put("components", components);
 
+        Calendar calendar = Calendar.getInstance();
+
+        dates.put( "dayOfWeek", calendar.getFirstDayOfWeek());
```

Comment:
Prettier should have fixed it
```suggestion
        dates.put("dayOfWeek", calendar.getFirstDayOfWeek());
```